### PR TITLE
Added News and Pages

### PR DESCRIPTION
**Pull request by iwillwen**

### CODE REVIEW

File: controllers/admin.js [hunk 1]
```diff
@@ -54,4 +56,171 @@ exports.viewTopic = function (req, res, next) {
...
+
+  var post = req.body;
+  var doc = {
+    title: post.title,
+    content: post.content
+  };
+
+  news.editNews(post.id, doc, function (err, record) {
+    if (err) {
+      res.send({'status': 'fail', 'message': 'unauthorized'}, 500);
```
Comment (JacksonTian) [inline controllers/admin.js]: 未授权的状态吗应该是401

File: controllers/admin.js [hunk 2]
```diff
@@ -54,4 +56,171 @@ exports.viewTopic = function (req, res, next) {
...
+  var doc = {
+    title: post.title,
+    sign: post.sign,
+    content: post.content,
+    language: req.locales[0]
+  };
+
+  pages.addPage(doc, function (err, page) {
+    if (err) {
+      res.send({'status': 'fail', 'message': 'unauthorized'}, 500);
```
Comment (JacksonTian) [inline controllers/admin.js]: 401

File: controllers/admin.js [hunk 3]
```diff
@@ -54,4 +56,171 @@ exports.viewTopic = function (req, res, next) {
...
+  var post = req.body;
+  var doc = {
+    title: post.title,
+    content: post.content,
+    date: new Date()
+  };
+
+  news.addNews(doc, function (err, news) {
+    if (err) {
+      res.send({'status': 'fail', 'message': 'unauthorized'}, 500);
```
Comment (JacksonTian) [inline controllers/admin.js]: 401

**Comment by iwillwen** [comment]
ok

**Pull request merged**